.github/workflows/CI.yml

```
@@ -35,3 +35,21 @@ jobs:
         continue-on-error: true
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
+  docker-deploy:
+    runs-on: ubuntu-latest
+    needs: test
+    env:
+      DOCKERHUB_PROJECT: gp-v2-contracts
+    steps:
+      - uses: actions/checkout@v2
+      - name: Dockerhub login
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_NAME }}
```

ubernit: I know we use this elsewhere but
```suggestion
          username: ${{ secrets.DOCKER_USER }}
```

:stuck_out_tongue:

---

Devops set the variable name and I don't know the secrets, so I had to changed the variable from `DOCKER_USER` to `DOCKER_NAME` to make it work. :slightly_smiling_face: 
Should I ask them to change it?

---

Nah, not worth the trouble in my opinion :stuck_out_tongue: